flytekit/clis/sdk_in_container/run.py

```diff
@@ -254,15 +256,13 @@ def convert_to_blob(
         value: typing.Union[Directory, FileParam],
     ) -> Literal:
         if isinstance(value, Directory):
-            uri = self.get_uri_for_dir(value)
+            uri = self.get_uri_for_dir(ctx, value)
         else:
             uri = value.filepath
             if self._remote and value.local:
                 fp = pathlib.Path(value.filepath)
-                md5, _ = script_mode.hash_file(value.filepath)
-                df_remote_location = self._create_upload_fn(filename=fp.name, content_md5=md5)
```

Comment:
is `_create_upload_fn` still being used anywhere?   can we remove it completely?